lib/components/user/monitored-trip/trip-summary-pane.tsx

```diff
@@ -1,61 +1,190 @@
+
+  @media (max-width: 600px) {
+    flex-direction: column;
+  }
+`
+
+const LocationDetails = styled.div`
+  align-items: start;
+  border-right: 1px solid #ddd;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  justify-content: center;
+  padding: 20px;
+  width: 50%;
+
+  @media (max-width: 600px) {
+    align-items: center;
+    border-bottom: 1px solid #ddd;
+    border-right: 0;
+    border-top: 1px solid #ddd;
+    width: 100%;
+  }
+`
+const ItineraryDetails = styled.div`
+  display: flex;
+  justify-content: center;
+  padding: 20px;
+  width: 50%;
+
+  @media (max-width: 600px) {
+    width: 100%;
+  }
+`
+const TextWIcon = styled.div`
+  align-items: center;
+  display: flex;
+  gap: 7px;
+  justify-content: left;
+  width: 250px;
+
+  svg {
+    flex-shrink: 0;
+  }
+`
+
+const TripDetailWithIcon = styled(TextWIcon)`
+  gap: 12px;
+
+  svg {
+    width: 16px;
+  }
+
+  &:last-of-type {
+    align-items: flex-start;
+
+    svg {
+      margin-top: 2px;
+    }
+  }
+`
+
+const TripDetailsList = styled.ul`
+  align-items: start;
+  display: flex;
+  flex-direction: column;
+  list-style: none;
+  gap: 15px;
+  justify-content: center;
+  padding: 0;
+`
+
+const ToggleNotificationButton = styled.button`
+  background: transparent;
+  border: none;
+  padding: 0;
+  text-decoration: underline;
+`
+
 const TripSummaryPane = ({
-  monitoredTrip
+  from,
+  handleTogglePauseMonitoring,
+  monitoredTrip,
+  pendingRequest,
+  to
 }: MonitoredTripProps): JSX.Element => {
   const { itinerary, leadTimeInMinutes } = monitoredTrip
-
   if (!itinerary) {
     return (
       <div>
         <FormattedMessage id="common.itineraryDescriptions.noItineraryToDisplay" />
       </div>
     )
   } else {
-    const days = (
-      <FormattedList
-        type="conjunction"
-        value={dayFieldsToArray(monitoredTrip).map((d) => (
-          <FormattedDayOfWeek day={d} key={d} />
-        ))}
-      />
-    )
+    const days = dayFieldsToArray(monitoredTrip)
+
+    const testHandle = () => {
+      if (handleTogglePauseMonitoring) {
+        handleTogglePauseMonitoring()
+      }
+    }
 
     return (
-      <>
+      <SavedTripBody>
         {/* TODO: use the modern itinerary summary built for trip comparison. */}
-        <TripSummary monitoredTrip={monitoredTrip} />
-        <p>
-          <FormattedMessage
-            id="components.TripSummaryPane.happensOnDays"
-            values={{ days, strong: Strong }}
-          />
-        </p>
-        <p>
-          {monitoredTrip.isActive ? (
-            <FormattedMessage
-              id="components.TripSummaryPane.notifications"
-              values={{ leadTimeInMinutes, strong: Strong }}
-            />
-          ) : (
-            <FormattedMessage
-              id="components.TripSummaryPane.notificationsDisabled"
-              values={{ leadTimeInMinutes, strong: Strong }}
-            />
-          )}
-        </p>
-      </>
+        <LocationDetails>
+          <TextWIcon>
```

Comment:
Could we use the existing `IconWithText` component or does it not support multiline text?

Reply:
The `IconWithText` component is a lot more rigid, and I wound up relying on flex attributes a lot. I'm open to updating `IconWithText` to work in both cases if that feels like something we want to do!

Reply:
Thanks for checking. It is fine to use new code if `IconWithText` is too rigid.